base/src/main/java/uk/ac/ebi/atlas/bioentity/geneset/GeneSetPropertyService.java

```diff
@@ -38,13 +38,13 @@ public GeneSetPropertyService(GoPoTrader goPoTermTrader,
     public Map<BioentityPropertyName, Set<String>> propertyValuesByType(String identifier) {
         identifier = identifier.toUpperCase();
         if (GeneSetUtil.matchesReactomeID(identifier)) {
```

Comment:
It's kind of surprising you didn't go for changing the signature of `propertyValuesByType` but it's not a problem.